README.md

```diff
@@ -0,0 +1,73 @@
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+[![Build Status](https://travis-ci.org/Stream-SQL-TCK/Stream-SQL-TCK.svg?branch=master)](https://travis-ci.org/julianhyde/Stream-SQL-TCK)
+
+# Stream SQL TCK
```

Comment:
Meta feedback: I think what's missing in the README is information on how other people can contribute and join the community around this project?  At least I would see it beneficial if community participation is actively encouraged (and guided).

Reply:
OK, I'll add that.